pages/common/git-add.md

```diff
@@ -9,28 +9,28 @@
 
 - Add all files (tracked and untracked):
 
-`git add -A`
+`git add --all`
 
 - Add all files in the current folder:
 
 `git add .`
 
 - Only add already tracked files:
 
-`git add -u`
+`git add --update`
 
 - Also add ignored files:
 
-`git add -f`
+`git add --force`
 
 - Interactively stage parts of files:
 
-`git add -p`
+`git add --patch`
 
 - Interactively stage parts of a given file:
 
-`git add -p {{path/to/file}}`
+`git add --patch {{path/to/file}}`
 
 - Interactively stage a file:
 
-`git add -i`
+`git add --interactive`
```

Comment:
The way I see it is that with brackets we can have all of the above. Longform for scripts, short form for everyday work and pasteability for convenience. It is just a matter of effort to mark them and nothing more. Like so:

```suggestion
- Add [A]ll files (tracked and untracked):

`git add --all`

- Add all files in the current folder:

`git add .`

- Only [u]pdate already tracked files:

`git add --update`

- Also add ignored files ([f]):

`git add --force`

- Interactively stage [p]arts of files:

`git add --patch`

- Interactively stage [p]arts of a given file:

`git add --patch {{path/to/file}}`

- [i]nteractively stage a file:

`git add --interactive`
```

Reply:
I'll leave this as my last attempt to persuade you on this subject. If I failed to convince you, I'll yield.

Reply:
Thank you for your work on this project. 

Regarding the current discussion, my opinion is that using the [a] notation in the documentation makes it untranslatable into Korean. 
I do agree, however, that **longform** is necessary for users who are unfamiliar with the commands. As for including **shortform** notations, I suggest adding them at the end of each explanation, like this:

```md
- Interactively stage a file [`--interactive` -> `-i`]:

`git add --interactive`
```

Reply:
In korean the solution would be to use ([a]) as was talked about earlier

Reply:
![obraz](https://github.com/user-attachments/assets/e536b09d-a036-48bf-8c15-865b35b6bb81)
IMO this would be a perfect solution, but this will need changes to clients, etc.
I still prefer the {{--long|-s}} notation but this suggestion with mnemonics is acceptable for me too,

Reply:
That would also be nice if it was user configurable which one it showed.

Reply:
> That would also be nice if it was user configurable which one it showed.

Yes, that would be cool. I think this and some other cool features could be implemented by having a .json or some other kind of additional file for each page, so this would break existing clients but will add extra features.

Reply:
For now I would like to hear opinions: Should I either go thought the commands and add bracket notation to the descriptions or the disjunction in the command?

Reply:
I prefer the ` {{--long|-s}}` option. Mnemonics do not blend well with the example descriptions often.